ts/screens/profile/ProfileMainScreen2.tsx

```diff
@@ -0,0 +1,152 @@
+import { connect } from "react-redux";
+import * as React from "react";
+import {
+  StyleSheet,
+} from "react-native";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import { List, View } from "native-base";
+import { SvgProps } from "react-native-svg";
+import ListItemComponent from "../../components/screens/ListItemComponent";
+import I18n from "../../i18n";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import { TranslationKeys } from "../../../locales/locales";
+import {
+  isLoggedIn,
+  isLoggedInWithSessionInfo
+} from "../../store/reducers/authentication";
+import { GlobalState } from "../../store/reducers/types";
+import { withLightModalContext } from "../../components/helpers/withLightModalContext";
+import ScreenContent from '../../components/screens/ScreenContent';
+import {
+  hasProfileEmailSelector,
+  isProfileEmailValidatedSelector,
+  profileEmailSelector,
+  profileNameSurnameSelector,
+  profileFiscalCodeSelector
+} from "../../store/reducers/profile";
+import NameSurnameIcon from "../../../img/assistance/nameSurname.svg";
+import FiscalCodeIcon from "../../../img/assistance/fiscalCode.svg";
+import EmailIcon from "../../../img/assistance/email.svg";
+
+type Props = ReturnType<typeof mapStateToProps>;
+
+const mapStateToProps = (state: GlobalState) => ({
+  sessionToken: isLoggedIn(state.authentication)
+    ? state.authentication.sessionToken
+    : undefined,
+  walletToken: isLoggedInWithSessionInfo(state.authentication)
+    ? state.authentication.sessionInfo.walletToken
+    : undefined,
+  profileEmail: profileEmailSelector(state),
+  isEmailValidated: isProfileEmailValidatedSelector(state),
+  hasProfileEmail: hasProfileEmailSelector(state),
+  nameSurname: profileNameSurnameSelector(state),
+  fiscalCode: profileFiscalCodeSelector(state)
+});
+
+export type ContextualHelpPropsMarkdown = {
+  title: TranslationKeys;
+  body: TranslationKeys;
+};
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "profile.main.contextualHelpTitle",
+  body: "profile.main.contextualHelpContent2"
+};
+
+const ProfileMainScreen2 = (props: Props) => {
+  const title = I18n.t("profile.main.title");
+  return (
+    <BaseScreenComponent
+      goBack={true}
+      headerTitle={title}
+      accessibilityLabel={title}
+      contextualHelpMarkdown={contextualHelpMarkdown}
+    >
+      <ScreenContent title={title} >
+        {screenContent(props)}
```

Comment:
if you rename `screenContent` to `ScreenContent` we than can use it as `<ScreenContent {...props} />`

Reply:
Wow. That's great! I'll just rename to `ProfileScreenContent` to avoid component name collision with our already present component with the same name.